Essentials/src/main/java/com/earth2me/essentials/commands/Commandcreatekit.java

```diff
@@ -60,9 +61,22 @@ public void run(final Server server, final User user, final String commandLabel,
         final String kitname = args[0];
         final ItemStack[] items = user.getBase().getInventory().getContents();
         final List<String> list = new ArrayList<>();
-        for (final ItemStack is : items) {
+
+        boolean usePaperSerial = ess.getSettings().isUseBetterKits();
```

Comment:
Perhaps rename this as it may not be unique to Paper forever? E.g. `useSerializationProvider`